server/database/queries/getDraftPosts.js

```diff
@@ -0,0 +1,19 @@
+const connection = require('../config/connection');
+
+getDraftEvents = (publisher_id) => connection.query(`
```

Comment:
Just an idea. You can make this query reusable by giving it two params: publisher_id, is_draft. Then, we can use it later to getPublishedEvents and getPublishedPublicServices

This is what I mean:
```javascript
getEvents = (publisher_id, is_Draft) => connection.query(`
SELECT id, category
  from event
  WHERE  is_draft = $2 and publisher_id = $1
`,
  [publisher_id, is_Draft]
);
```

The same goes for the other query.
This way, we don't need to make another two queries to get published posts